products/veeam-backup-and-replication

```diff
@@ -0,0 +1,123 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backup-and-replication
+alternate_urls:
+-   /veeam-backup
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-01-30
```

Comment:
Should we use RTM of GA dates ? I would feel more natural to me to use the GA date, but I am not a Veeam user so I don't now.